### PR TITLE
Automated PyPI deploy with Github Actions

### LINKED ISSUE #36: Deploying to PyPI with Github Actions
Deploying to PyPI with GitHub Actions:
https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

### PR DESCRIPTION
**Pull request by camilamaia**
## Description

Automated PyPI deploy with Github Actions

Reference: https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

Closes #36 

We can test it properly in the next release

### CODE REVIEW

**Comment by camilamaia** [comment]
We need to setup a `pyproject.toml` file 😞

File: .github/workflows/publish-to-pypi.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          --source
+          --binary
+          --out-dir dist/
+          .
+      - name: Publish distribution 📦 to PyPI
+        if: startsWith(github.event.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.pypi_password }}
```
Comment (barbosa) [inline .github/workflows/publish-to-pypi.yml:32]: Do we want to publish to Test Pypi too?

Comment (camilamaia) [reply .github/workflows/publish-to-pypi.yml:32]: Hmmm good point, I was lazy and ignored it 😆 

It makes sense to have it. But I believe in a step before, right? I mean, not in the release process. Cause then we can test it before sending to "production".

Comment (barbosa) [reply .github/workflows/publish-to-pypi.yml:32]: That's correct. We'd need another workflow.

Comment (camilamaia) [reply .github/workflows/publish-to-pypi.yml:32]: @barbosa I created a issue for that https://github.com/scanapi/scanapi/issues/146

**Pull request merged**